grpc-api/src/main/java/io/deephaven/grpc_api/runner/DeephavenApiServer.java

```diff
@@ -80,7 +80,7 @@ public static void startMain(PrintStream out, PrintStream err)
 
         // Close outstanding sessions to give any gRPCs closure.
         ProcessEnvironment.getGlobalShutdownManager().registerTask(ShutdownManager.OrderingCategory.MIDDLE,
-                sessionService::closeAllSessions);
+                sessionService::onShutdown);
```

Comment:
This suggests to me that onShutdown should be a no-op if we've hit onFatalError.

Reply:
Never mind, I see what you did here.